### ISSUE TITLE
Error code 403 when trying to download bundle

### ISSUE THREAD

**Issue by legojesus** [caiiiycuk/js-dos.com#253]
Seems like things are broken currently: 

1. npx create-dosbox app
2. type in any game's name (e.g. "Doom")
3. Select a game from the list of results
4. Get a 403 status error code. 

Found that the same error exists on the website: https://dos.zone/studio/?url=https%3A%2F%2Fdoszone-uploads.s3.dualstack.eu-central-1.amazonaws.com%2Fcustom/dos/doom.jsdos

I think the access to that AWS bucket has been revoked or restricted in a way that breaks things. 

Full error: 
```
? Please select the game from list DOOM

Downloading bundle

Error: Request failed with status code 403

    at RedirectableRequest.<anonymous> (/root/.npm/_npx/1/lib/node_modules/create-dosbox/node_modules/nodejs-file-downloader/makeRequest.js:20:31)

    at RedirectableRequest.emit (events.js:327:22)

    at RedirectableRequest._processResponse (/root/.npm/_npx/1/lib/node_modules/create-dosbox/node_modules/follow-redirects/index.js:356:10)

    at ClientRequest.RedirectableRequest._onNativeResponse (/root/.npm/_npx/1/lib/node_modules/create-dosbox/node_modules/follow-redirects/index.js:62:10)

    at Object.onceWrapper (events.js:422:26)

    at ClientRequest.emit (events.js:315:20)

    at HTTPParser.parserOnIncomingClient [as onIncoming] (_http_client.js:641:27)

    at HTTPParser.parserOnHeadersComplete (_http_common.js:126:17)

    at TLSSocket.socketOnData (_http_client.js:509:22)

    at TLSSocket.emit (events.js:315:20) {

  statusCode: 403events.js:315:20)

    at HTTPParser.parserOnIncomingClient [as onIncoming] (_http_client.js:641:27)

    at HTTPParser.parserOnHeadersComplete (_http_common.js:126:17)

    at TLSSocket.socketOnData (_http_client.js:509:22)

    at TLSSocket.emit (events.js:315:20) {

  statusCode: 403
}
```

**Comment by legojesus** [comment]
Ah, just saw that you're having problems paying AWS due to sanctions on Russia, which is why there's no access to the storage bucket.

**Comment by caiiiycuk** [comment]
Thank you, for reporting!

Fixed in v7.4.6, use it like this:
```
npx create-dosbox@latest my-app
```

**Comment by legojesus** [comment]
Thank you kindly @caiiiycuk ! Works great now :)
